packages/blocks/src/api/matchers.js

```diff
@@ -41,3 +46,10 @@ export function html( selector, multilineTag ) {
 		return match.innerHTML;
 	};
 }
+
+export const richText = ( selector, preserveWhiteSpace ) => ( el ) => {
+	const target = selector ? el.querySelector( selector ) : el;
+	return target
+		? RichTextData.fromHTMLElement( target, { preserveWhiteSpace } )
+		: new RichTextData();
```

Comment:
Should we just for easy of use allow `null` to be passed to `RichTextData.fromHTMLElement`, or is it better to be more explicit?

Reply:
Why is that a static empty method better than creating a new instance directly?

Reply:
it's not better, just different, and provides the ability to return a failure without crashing. I was also thinking about the purely superficial symmetry of `RTD.fromHTML` and `RTD.empty()` vs `RTD.fromHTML` and `new RTD()`